code/modules/clothing/spacesuits/plasmamen.dm

```diff
@@ -12,28 +12,84 @@
 	max_heat_protection_temperature = SPACE_SUIT_MAX_TEMP_PROTECT
 	species_restricted = list("Plasmaman")
 	flags = STOPSPRESSUREDMAGE
+	autoextinguish = 1
 
 	icon_state = "plasmaman_suit"
 	item_state = "plasmaman_suit"
 
+	actions_types = list(/datum/action/item_action/eject_depleted_cartridges)
```

Comment:
Bummer. Boring and simple it is then, I guess. I thought it was quite straightforward, there really isn't that much complexity to the way the suit reloading was handled.

It was fun getting this all to work and I felt it'd make the suits more interesting and legitimate.
Not to mention that it means that these cartridges could be refilled from water canisters instead of being one-offs.

The simple cartridges way makes the suits feel cheapened.
*shrug

I'd really rather not, but I don't suppose I've an option